operate/webapp/src/main/java/io/camunda/operate/webapp/backup/BackupRepositoryWrapper.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.backup;
+
+import io.camunda.operate.exceptions.OperateElasticsearchConnectionException;
+import io.camunda.operate.exceptions.OperateOpensearchConnectionException;
+import io.camunda.operate.exceptions.OperateRuntimeException;
+import io.camunda.operate.webapp.elasticsearch.backup.ElasticsearchBackupRepository;
+import io.camunda.operate.webapp.opensearch.backup.OpensearchBackupRepository;
+import io.camunda.webapps.backup.BackupRepository;
+import io.camunda.webapps.backup.BackupService.SnapshotRequest;
+import io.camunda.webapps.backup.GetBackupStateResponseDto;
+import io.camunda.webapps.backup.exceptions.InvalidRequestException;
+import io.camunda.webapps.backup.exceptions.ResourceNotFoundException;
+import io.camunda.webapps.backup.repository.BackupRepositoryConnectionException;
+import io.camunda.webapps.backup.repository.GenericBackupException;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Wraps a BackupRepository in order to map the exception thrown by the generic implementation to
```

Comment:
:wrench: As it confused me, let's document this is expected to be removed in a bit, in case someone stumbles on it during its intermediate state.